package.json

```diff
@@ -27,9 +27,9 @@
     "google-libphonenumber": "^2.0.15",
     "jquery": "^3.2.1",
     "medic-enketo-xslt": "^1.1.1",
+    "medic-nootils": "^1.0.3",
     "messageformat": "^1.0.0",
     "moment": "^2.17.1",
-    "nools": "^0.4.4",
```

Comment:
I wonder if webapp should keep a direct dependency on nools? It probably doesn't matter, but it feels weird to depend on the transitive dependency.